pjt00/tweet/templates/tweet/profile.html

```diff
@@ -0,0 +1,33 @@
+{% extends "tweet/base.html" %}
+{% block title %}{{ user }} -{{ block.super }}{% endblock %}
+{% block content %}
+<h1>{{ user }} さんのツイート</h1>
+{% if user.id is not request.user.id %}
+  {% if not ff %}
+  <p>
+    <form name='follow' action="{% url 'tclone:follow' pk=user.pk %}" method="POST">
+      フォローしていません！
+      {% csrf_token %}
+      {{ form.as_p }}
+      <button type="submit" onclick='return confirm("フォローしますか？");'>フォロー</button>
+    </form>
+  </p>
+  {% else %}
+  <p>
+    <form class="delete_confirm" action="{% url 'tclone:followx' pk=user.pk %}" method="post">
+      フォローしています！
+      {% csrf_token %}
+      <button type="submit" onclick='return confirm("削除しますか？");'>フォロー解除</button>
+    </form>
+  </p>
+  {% endif %}
+{% else %}
+    <p>
+      あなたのツイートが表示されています！
+    </p>
+{% endif %}
+{% for tweet in tweets reversed %}
+  <em><a href="{% url 'tweet:profile' pk=tweet.user.pk %}">{{ tweet.user }}</a></em> / <a href="{% url 'tweet:tdetail' pk=tweet.pk %}">詳細</a>
+  <p>{{ tweet.tweet }}</p>

```

Comment:
Tweetモデルの`__str__`メソッドを確認しましょう。
```suggestion
  <p>{{ tweet }}</p>

```

Reply:
[**4.0.2**]で修正しました。`{{ tweet }}`がpythonでいう`str(tweet)`となるから、`def __str__`によって`tweet.tweet`が返ってくるのでこれで良いということでしょうか。

Reply:
そうです。